src/ca65/scanner.c

```diff
@@ -132,163 +132,164 @@ struct DotKeyword {
-    { ".REFERENCED",    TOK_REFERENCED  },
-    { ".RELOC",         TOK_RELOC       },
-    { ".REPEAT",        TOK_REPEAT      },
-    { ".RES",           TOK_RES         },
-    { ".RIGHT",         TOK_RIGHT       },
-    { ".RODATA",        TOK_RODATA      },
-    { ".SCOPE",         TOK_SCOPE       },
-    { ".SEGMENT",       TOK_SEGMENT     },
-    { ".SET",           TOK_SET         },
-    { ".SETCPU",        TOK_SETCPU      },
-    { ".SHL",           TOK_SHL         },
-    { ".SHR",           TOK_SHR         },
-    { ".SIZEOF",        TOK_SIZEOF      },
-    { ".SMART",         TOK_SMART       },
-    { ".SPRINTF",       TOK_SPRINTF     },
-    { ".STRAT",         TOK_STRAT       },
-    { ".STRING",        TOK_STRING      },
-    { ".STRLEN",        TOK_STRLEN      },
-    { ".STRUCT",        TOK_STRUCT      },
-    { ".TAG",           TOK_TAG         },
-    { ".TCOUNT",        TOK_TCOUNT      },
-    { ".TIME",          TOK_TIME        },
-    { ".UNDEF",         TOK_UNDEF       },
-    { ".UNDEFINE",      TOK_UNDEF       },
-    { ".UNION",         TOK_UNION       },
-    { ".VERSION",       TOK_VERSION     },
-    { ".WARNING",       TOK_WARNING     },
-    { ".WORD",          TOK_WORD        },
-    { ".XMATCH",        TOK_XMATCH      },
-    { ".XOR",           TOK_BOOLXOR     },
-    { ".ZEROPAGE",      TOK_ZEROPAGE    },
+    { ".A16",           TOK_A16          },
+    { ".A8",            TOK_A8           },
+    { ".ADDR",          TOK_ADDR         },
+    { ".ADDRSIZE",      TOK_ADDRSIZE     },
+    { ".ALIGN",         TOK_ALIGN        },
+    { ".AND",           TOK_BOOLAND      },
+    { ".ASCIIZ",        TOK_ASCIIZ       },
+    { ".ASSERT",        TOK_ASSERT       },
+    { ".AUTOIMPORT",    TOK_AUTOIMPORT   },
+    { ".BANK",          TOK_BANK         },
+    { ".BANKBYTE",      TOK_BANKBYTE     },
+    { ".BANKBYTES",     TOK_BANKBYTES    },
+    { ".BITAND",        TOK_AND          },
+    { ".BITNOT",        TOK_NOT          },
+    { ".BITOR",         TOK_OR           },
+    { ".BITXOR",        TOK_XOR          },
+    { ".BLANK",         TOK_BLANK        },
+    { ".BSS",           TOK_BSS          },
+    { ".BYT",           TOK_BYTE         },
+    { ".BYTE",          TOK_BYTE         },
+    { ".CASE",          TOK_CASE         },
+    { ".CHARMAP",       TOK_CHARMAP      },
+    { ".CODE",          TOK_CODE         },
+    { ".CONCAT",        TOK_CONCAT       },
+    { ".CONDES",        TOK_CONDES       },
+    { ".CONST",         TOK_CONST        },
+    { ".CONSTRUCTOR",   TOK_CONSTRUCTOR  },
+    { ".CPU",           TOK_CPU          },
+    { ".DATA",          TOK_DATA         },
+    { ".DBG",           TOK_DBG          },
+    { ".DBYT",          TOK_DBYT         },
+    { ".DEBUGINFO",     TOK_DEBUGINFO    },
+    { ".DEF",           TOK_DEFINED      },
+    { ".DEFINE",        TOK_DEFINE       },
+    { ".DEFINED",       TOK_DEFINED      },
+    { ".DEFINEDMACRO",  TOK_DEFINEDMACRO },
```

Comment:
I suggest that, after the rebase, you move the right edge (the column of right braces) of this initiator over to the next 8-character tab stop (column 49).  Then, if anyone adds a longer token to this list, the other lines might not need to be changed.